discovery-provider/src/tasks/entity_manager/entities/playlist.py

```diff
@@ -87,8 +86,8 @@ def update_playlist_routes_table(
     new_playlist_route.owner_id = playlist_record.playlist_owner_id
     new_playlist_route.playlist_id = playlist_record.playlist_id
     new_playlist_route.is_current = True
-    new_playlist_route.blockhash = playlist_record.blockhash
-    new_playlist_route.blocknumber = playlist_record.blocknumber
+    new_playlist_route.blockhash = cast(str, playlist_record.blockhash)
```

Comment:
casts are scary, what type were these before?

Reply:
Fixed by typing the FK in the model more correctly